main.go

```diff
@@ -49,13 +51,20 @@ func generatorsHelp() string {
 	return buffer.String()
 }
 
+func isPipe() bool {
+	stat, _ := os.Stdin.Stat()
```

Comment:
why wouldn't we check the error here? (Honest question, I don't know the answer)